src/api/BaseUrl.js

```diff
@@ -1,10 +1,10 @@
 const baseUrl = 'https://bootcamp-api.codeit.kr/api';

```

Comment:
P1:
상수값은 대문자로 적어주시면 좋겠습니다.

```suggestion
const BASE_URL = 'https://bootcamp-api.codeit.kr/api';
```